netweaver/install_aas.sls

```diff
@@ -28,6 +29,16 @@ create_aas_inifile_{{ instance_name }}:
         hana_password: {{ netweaver.hana.password }}
         hana_inst: {{ hana_instance }}
 
+{% if node.extra_parameters is defined %}
+update_aas_inifile_{{ instance_name }}:
+  module.run:
+    - netweaver.update_conf_file:
+      - conf_file: {{ inifile }}
+      - {%- for key,value in node.extra_parameters.items() %}
```

Comment:
Will the `-` hypen also not come at end, ie before `%}`

Reply:
LGTM. See: https://jinja.palletsprojects.com/en/master/templates/#whitespace-control

Reply:
No, the `-` only comes at the beginning.

Reply:
yes we use already this `-` in begin only in other projects